pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry>=1.0.5"]
+requires = ["poetry-core>=1.0.5"]
 build-backend = "poetry.masonry.api"
```

Comment:
presumably it's for some reason not broken, but I'd expect that you should want "poetry.core.masonry.api" after making this change